Makefile.kubebuilder

```diff
@@ -0,0 +1,127 @@
+
+.PHONY: fmt
+fmt: ## Run go fmt against code.
+	go fmt ./...
+
+.PHONY: vet
+vet: ## Run go vet against code.
+	go vet ./...
+
+##@ Build
+
+.PHONY: kubebuilder-build
+kubebuilder-build: generate fmt vet ## Build manager binary.
+	go build -o bin/manager ./cmd/reposervercontroller/main.go
+
+.PHONY: kubebuilder-run
+kubebuilder-run: manifests generate fmt vet ## Run a controller from your host.
+	go run ./cmd/reposervercontroller/main.go
+
+# If you wish built the manager image targeting other platforms you can use the --platform flag.
+# (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
+# More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+.PHONY: docker-build
+docker-build: kubebuilder-test ## Build docker image with the manager.
+	docker build -t ${IMG} docker/repositoryserver-controller
+
+.PHONY: docker-push
+docker-push: ## Push docker image with the manager.
+	docker push ${IMG}
+
+# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
+# architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
+# - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
+# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/amd64
+.PHONY: docker-buildx
+docker-buildx: kubebuilder-test ## Build and push docker image for the manager for cross-platform support
+	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' docker/repositoryserver-controller/Dockerfile > docker/repositoryserver-controller/Dockerfile.cross
+	- docker buildx create --name project-v3-builder
+	docker buildx use project-v3-builder
+	- docker buildx build --push --platform=$(PLATFORMS) --tag ${IMG} -f docker/repositoryserver-controller/Dockerfile.cross
+	- docker buildx rm project-v3-builder
+	rm docker/repositoryserver-controller/Dockerfile.cross
+
+##@ Deployment
+
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
+.PHONY: install
+install: ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	kubectl apply -f pkg/customresource/repositoryserver.yaml
+
+.PHONY: uninstall
+uninstall: ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
+	kubectl delete -f pkg/customresource/repositoryserver.yaml
+
```

Comment:
If we are packaging this to helm chart, should we remove these target?

Reply:
No, lets have it for development purposes